cypress/integration/data-browser/dataset-filtering.spec.js

```diff
@@ -0,0 +1,201 @@
+import {
+  openSelector,
+  dbURL,
+  isBeta,
+  isProd,
+  waitUpto2Mins,
+} from '../../support/helpers'
+
+const YEARS = [2021, 2020, 2019, 2018]
+const { HOST, ENVIRONMENT } = Cypress.env()
+const dbUrl = dbURL.bind(null, HOST)
+
+describe(`Data Browser - Dataset Filtering`, function () {
```

Comment:
newer standards is to have arrow functions now right? think keyword function is only useful for prototypal functionalities now.